tests/bgp/test_bgp_azng_migration.py

```diff
@@ -131,13 +149,31 @@ def test_bgp_azng_migration(duthosts, enum_upstream_dut_hostname):
             res = duthost.shell(duthost.get_vtysh_cmd_for_namespace(bgp_nbr_recv_cmd, peer_device_namespace))
             routes_json = json.loads(res['stdout'])
             assert routes_json['totalPrefixCounter'] > 0
```

Comment:
There is an assert without failed reason too, can you just search "assert " in the script to make sure you didn't miss any places else.